chart/compass/values.yaml

```diff
@@ -153,6 +153,15 @@ global:
       namespace: "compass-system"
     proxy:
       port: 15020
+    namespace: istio-system
+    ingressgateway:
+      workloadLabel: istio-ingressgateway
+      correlationHeaderRewriteFilter:
+        expectedHeaders:
+        - "x-request-id"
```

Comment:
The headers entering the filter are all lowercased

Reply:
this might be a bit of a stretch but i remember for cf there was some vcap id or something, that was passed. Should we consider it as well ?